deployments/jsonnetfile.json

```diff
@@ -8,6 +8,15 @@
         }
       },
       "version": ""
+    },
+    {
```

Comment:
this shouldn't be here, only in the `jsonnetfile.json` that the local dependency points to

Reply:
Ah, I couldn't find the docs anywhere. Could you link me to that? I'll add a README how to set it up and generate manifests.

Reply:
What exactly where you looking for in the docs? Dependencies resolve transitively, so by seeing the local dependency, `jb` will resolve the transitive dependencies by reading all dependencies from the dependencies' `jsonnetfile.json` (and then does this recursively). Which means ksonnet should already be installed if you run `jb install` in the `deployments/` directory as it reads [this jsonnetfile.json](https://github.com/conprof/conprof/blob/master/deployments/jsonnet/conprof/jsonnetfile.json).

Reply:
installing ksonnet so that our makefile picks it up, as right now I can see that our makefile looks for stuff that's in `vendor`. So how do I configure ksonnet so that it's not in vendor and imported correctly?

 Ref: https://github.com/conprof/conprof/blob/master/deployments/Makefile#L6 

Install instructions in ksonnet only say how to install their cli https://github.com/ksonnet/ksonnet/blob/master/docs/build-install.md

Reply:
Aha I see!

We should add a `vendor` make target, which depends on `jsonnetfile.lock.json`, and runs `jb install` in the `deployments/` directory. That should install things as expected (and update when needed).